From 59e9d1863924fee539873422741fdb9596a71fb5 Mon Sep 17 00:00:00 2001
From: Joe Zheng <joe.zheng@intel.com>
Date: Mon, 17 Jun 2019 10:52:20 +0800
Subject: [PATCH] Enable overlayfs support for SELinux

to fix the issue:
Could not set context for /init: Operation not supported on transport endpoint

Change-Id: I269fa3a9e2018ae44894f44d0dee10bc30b3f357
Topic: aic.selinux
Tracked-On: ACP-600
(cherry picked from commit 97360a1b890f5e9b338c4544c3fb92b0c7697752)
---
 prebuilts/api/28.0/private/fs_use | 1 +
 private/fs_use                    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/prebuilts/api/28.0/private/fs_use b/prebuilts/api/28.0/private/fs_use
index 4bd11126..5445c679 100644
--- a/prebuilts/api/28.0/private/fs_use
+++ b/prebuilts/api/28.0/private/fs_use
@@ -8,6 +8,7 @@ fs_use_xattr xfs u:object_r:labeledfs:s0;
 fs_use_xattr btrfs u:object_r:labeledfs:s0;
 fs_use_xattr f2fs u:object_r:labeledfs:s0;
 fs_use_xattr squashfs u:object_r:labeledfs:s0;
+fs_use_xattr overlay u:object_r:labeledfs:s0;
 
 # Label inodes from task label.
 fs_use_task pipefs u:object_r:pipefs:s0;
diff --git a/private/fs_use b/private/fs_use
index 4bd11126..5445c679 100644
--- a/private/fs_use
+++ b/private/fs_use
@@ -8,6 +8,7 @@ fs_use_xattr xfs u:object_r:labeledfs:s0;
 fs_use_xattr btrfs u:object_r:labeledfs:s0;
 fs_use_xattr f2fs u:object_r:labeledfs:s0;
 fs_use_xattr squashfs u:object_r:labeledfs:s0;
+fs_use_xattr overlay u:object_r:labeledfs:s0;
 
 # Label inodes from task label.
 fs_use_task pipefs u:object_r:pipefs:s0;
-- 
2.21.0

